### PR TITLE
Cv 684 levy status

### PR DESCRIPTION
**Pull request by Pauljgraham**

### CODE REVIEW

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web.UnitTests/Controllers/CohortControllerTests/WhenIPostDraftApprenticeshipViewModel.cs [hunk 1]
```diff
@@ -14,7 +14,7 @@
 using SFA.DAS.ProviderUrlHelper;
 using RedirectResult = Microsoft.AspNetCore.Mvc.RedirectResult;
 
-namespace SFA.DAS.ProviderCommitments.Web.UnitTests.Controllers.CreateCohortWithDraftApprenticeshipControllerTests
+namespace SFA.DAS.ProviderCommitments.Web.UnitTests.Controllers.CohortControllerTests
```
Comment (chrisfoster76) [inline src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web.UnitTests/Controllers/CohortControllerTests/WhenIPostDraftApprenticeshipViewModel.cs:17]: 👍

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/Cohort/AddDraftApprenticeshipViewModelMapper.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+using SFA.DAS.CommitmentsV2.Shared.Models;
+
+namespace SFA.DAS.ProviderCommitments.Web.Mappers.Cohort
+{
+    public class AddDraftApprenticeshipViewModelMapper : IMapper<CreateCohortWithDraftApprenticeshipRequest, AddDraftApprenticeshipViewModel>
+    {
+        private readonly ICommitmentsApiClient _commitmentsApiClient;
+        private readonly IMediator _mediator;
+
+
```
Comment (chrisfoster76) [inline src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/Cohort/AddDraftApprenticeshipViewModelMapper.cs:18]: Presumably this space is for ventilation?

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Models/AddDraftApprenticeshipViewModel.cs [hunk 3]
```diff
@@ -6,6 +6,6 @@ public class AddDraftApprenticeshipViewModel : DraftApprenticeshipViewModel, IAu
     {
         public string EmployerAccountLegalEntityPublicHashedId { get; set; }
         
-        public long? AccountLegalEntityId { get; set; }
+        public long AccountLegalEntityId { get; set; }
```
Comment (chrisfoster76) [inline src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Models/AddDraftApprenticeshipViewModel.cs:9]: 👍

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/Cohort/AddDraftApprenticeshipViewModelMapper.cs (hunk 2 above)
Comment (Pauljgraham) [reply src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/Cohort/AddDraftApprenticeshipViewModelMapper.cs:18]: I do try to find these blank lines, before I submit a PR. I'm convinced someone else is adding them

**Pull request merged**